components/dashboard/src/workspaces/WorkspaceEntry.tsx

```diff
@@ -206,7 +206,7 @@ export function WorkspaceEntry({ desc, model, isAdmin, stopWorkspace }: Props) {
                 </div>
             </ItemField>
             <ItemField className="w-2/12 flex my-auto">
-                <Tooltip content={`Created ${dayjs(desc.workspace.creationTime).fromNow()}`}>
+                <Tooltip content={dayjs(desc.workspace.creationTime).format("MMM D, YYYY")}>
```

Comment:
**question**: Is this working on the workspaces list? Seems like the tooltip duplicates what's on the date below.

| Workspaces |
|-|
| <img width="1440" alt="Screenshot 2023-01-24 at 12 11 39 AM (2)" src="https://user-images.githubusercontent.com/120486/214161478-ac2539d0-2bd0-4d08-9abd-b1641ea14457.png"> |

Reply:
Looks good—resolving.